code/src/pgsql_io/pgsql_io.mli

```diff
@@ -192,6 +192,43 @@ module Prepared_stmt : sig
     'q
 end
 
+module Auth_scram : sig
```

Comment:
All of this is internal functionality doesn't not need to be exposed as part of the pgsql API.  If we want to write tests for some of the SCRAM stuff, I would recommend making a scram library and moving the specific SCRAM functions to it rather than exposing it as part of the pgsql library.  As things stand now, I don't think we have to do that, so we can remove this from the interface.